test/integration/mocha-qunit-sauce.js

```diff
@@ -20,17 +20,12 @@ test('mocha-qunit - sauce', function (t) {
   var zuul = Zuul(config)
 
   getBrowsers(function (err, allBrowsers) {
+    t.error(err, 'no error')
+
     var browsers = flattenBrowser(browsersToTest, allBrowsers)
     browsers.forEach(zuul.browser.bind(zuul))
 
-    t.plan(browsers.length * 3 + 3)
```

Comment:
@vweevers I originally thought there was something wrong with `t.plan()` and that's why the tests hung. We might want to put this back now.